js/index.js

```diff
@@ -0,0 +1,202 @@
+/* eslint-disable camelcase */
+import { SlideDeck } from './slidedeck.js';
+
+const map = L.map('map', { scrollWheelZoom: false }).setView([0, 0], 0);
+const mapboxKey = 'pk.eyJ1IjoiY2xhdWRsb3ciLCJhIjoiY20weTY3MDZoMDNocTJrbXpqa3lqZWJlaSJ9.3N1iXpEvsJ0GwajGVwwkTg';
+const mapboxStyle1 = 'mapbox/light-v11';
+
+// ## The Base Tile Layer
+const baseTileLayer = L.tileLayer(`https://api.mapbox.com/styles/v1/${mapboxStyle1}/tiles/512/{z}/{x}/{y}{r}?access_token=${mapboxKey}`, {
+  maxZoom: 16,
```

Comment:
**suggestion:** When using Mapbox's 512px tiles, you should usually set up your tile layer like this. This will reduce the number of tile requests your map has to make to Mapbox, and will prevent your map labels from showing up super tiny.
```suggestion
  maxZoom: 16,
  tileSize: 512,
  zoomOffset: -1,
```
What's happening right now on your map is Mapbox's 512 pixel tiles are being squeezed into Leaflet's default 256x256 pixel tile size, so each tile is getting scaled down to half it's size in each direction. This code is telling leaflet to use 512 pixel tiles instead, but use the tiles from one zoom level offset from what leaflet might expect (i.e. request the zoom level 7 tiles from Mapbox when you're at zoom level 8 on the map).